numpy/linalg/tests/test_linalg.py

```diff
@@ -2043,10 +2043,9 @@ def test_tensorinv_result(self):
 
 def test_unsupported_commontype():
     # linalg gracefully handles unsupported type
-    arr = np.array([[1, -2], [2, 5]], dtype='float16')
+    arr = np.array(['3.5', '2.'], dtype='float16')
```

Comment:
What is the motivation behind the change in the array values? Either one gives an array with a float16 dtype and the string values seem strange to me.

Reply:
@rossbar this is indeed a trivial change, which leads to a trivial but interesting argument, anyway I will close this for PR mangement.